exonum-java-binding-common/src/main/java/com/exonum/binding/common/serialization/StandardSerializers.java

```diff
@@ -63,10 +74,21 @@
   }
 
   /**
-   * Returns a serializer of longs using variable length encoding in little-endian byte order.
+   * Returns a serializer of unsigned longs using variable length encoding.
+   * These more efficiently encodes values for the range {@code [0; 2^49-1]}
+   * than {@link StandardSerializers#fixed64()}.
    */
-  public static Serializer<Long> varInt64() {
-    return VarInt64Serializer.INSTANCE;
+  public static Serializer<Long> uint64() {
```

Comment:
Would you please use this in our QA service (for the `com.exonum.binding.qaservice.QaSchema#counters`)?